Formula/postgresql@14.rb

```diff
@@ -145,8 +145,10 @@ def caveats
 
         You can migrate to a versioned data directory by running:
           mv -v "#{old_postgres_data_dir}" "#{var/name}"
+          sed -i.bak "s|<string>#{old_postgres_data_dir}</string>|<string>#{var/name}</string>|" #{opt_prefix}/homebrew.mxcl.postgresql@14.plist
+          sed -i.bak "s|-D #{old_postgres_data_dir}$|-D #{var/name}|" #{opt_prefix}/homebrew.postgresql@14.service
```

Comment:
I probably need advice regarding `homebrew.mxcl.postgresql@14.plist` location.
In the reported issue (https://github.com/Homebrew/homebrew-core/issues/109644) it's `~/Library/LaunchAgents/homebrew.mxcl.postgresql@14.plist`, on my local machine it's `#{opt_prefix}/homebrew.mxcl.postgresql@14.plist`.
What is the logic here?

Reply:
I think it gets put in `~/Library/LaunchAgents` if you do `brew services start`.

I think stopping it and restarting it will replace the one in `~/Library/LaunchAgents` with the one from `prefix`, but I could be wrong. @SMillerDev would know.

Reply:
Good to know!

Reply:
AFAIR a service stop and start will reinstall the service from the prefix.